src/pages/Sign/sections/SignUp.tsx

```diff
@@ -121,79 +149,137 @@ const UserSchema = Yup.object().shape({
+                    }}
+                  >
+                    {(title) => title}
+                  </TranslationsPlaceholderConsumer>
+                )}
+              </OfferContainer>
+            </Header>
+          </HeaderWrapper>
+          <SessionContainer>
+            {(token) =>
+              token ? (
+                <>
+                  <Mutation<boolean, SignOfferMutationVariables>
+                    mutation={SIGN_MUTATION}
+                  >
+                    {(signOffer, signProps) => (
+                      <Formik
+                        initialValues={{
+                          email: '',
+                          personalNumber: '',
+                        }}
+                        validationSchema={userSchema}
+                        onSubmit={(values) =>
+                          signOffer({
+                            variables: {
+                              personalNumber: values.personalNumber,
+                              email: values.email,
+                            },
+                          })
+                        }
+                      >
+                        {({ errors, touched }) => (
+                          <CustomForm>
+                            <InputTitle>
+                              <TranslationsConsumer textKey="SIGN_INPUT_ONE_TITLE">
+                                {(title) => title}
+                              </TranslationsConsumer>
+                            </InputTitle>
+
+                            <InputField
+                              name="email"
+                              id="email"
+                              style={{
+                                borderBottom: touched.email
+                                  ? errors.email
+                                    ? '3px solid red'
+                                    : [`3px solid ${colors.GREEN}`]
+                                  : [`3px solid ${colors.DARK_PURPLE}`],
+                              }}
+                            />
+                            {errors.email && touched.email ? (
+                              <ErrorMessage>
+                                <TranslationsConsumer textKey={errors.email}>
+                                  {(errorMessage) => errorMessage}
+                                </TranslationsConsumer>
+                              </ErrorMessage>
+                            ) : null}
+                            <InputTitle>
+                              <TranslationsConsumer textKey="SIGN_INPUT_TWO_TITLE">
+                                {(title) => title}
+                              </TranslationsConsumer>
+                            </InputTitle>
+                            <InputField
+                              name="personalNumber"
+                              id="personalNumber"
+                              style={{
+                                borderBottom: touched.personalNumber
```

Comment:
...also we're kinda repeating ourselves. We should definitely move this out to a utils function